src/3d-force-graph.js

```diff
@@ -313,9 +318,8 @@ export default Kapsule({
       });
 
     //
-
     // Kick-off renderer
-    (function animate() { // IIFE
+    (state.animate = function animate() { // IIFE
```

Comment:
If this is being added to the state, let's use a more descriptive name, like `animateCycle` or so.